lib/Windowing/InViewList.js

```diff
@@ -0,0 +1,67 @@
+import React, { useContext, useEffect, useState } from 'react';
+import { arrayOf, func, number, shape } from 'prop-types';
+import { WindowingContext } from './WindowingProvider';
+
+function InViewList({ items, offset, children, itemHeight }) {
+  const [renderIndexes, setRenderIndexes] = useState({});
+  const { scrollTop, height } = useContext(WindowingContext);
+
+  const subtractFromStartIndex = index =>
+    index - offset <= 0 ? 0 : index - offset;
```

Comment:
This is the same as 
```suggestion
    Math.max(index - offset, 0);
```
which you could probably just stick on line 22